app/viewmodels/SelectItemsTableHelper.scala

```diff
@@ -48,7 +48,7 @@ class SelectItemsTableHelper @Inject()(link: link, list: list) {
             classes = "white-space-nowrap"
           ),
           TableRow(
-            content = Text(messages(cnCodeInformation.cnCodeDescription)),
+            content = Text(item.commercialDescription.getOrElse("")),
```

Comment:
Would it be worth defaulting to the `cnCodeDescription` if the commercial description doesn't exist? At least it would always have something then.

Reply:
Being discussed within the team

Reply:
@MJCallahanPage Pushed another commit based on the discussion had.